### ISSUE TITLE
Anbox loads but can't click anything on the dashboard 

### ISSUE THREAD

**Issue by lokii86** [anbox/anbox#1121]
So I'm sorry if this has been posted already but I couldn't find this issue posted already.  

I had 0 issues installing anbox installed my desired apk files and then when launching anbox ro run apps I cant select or run anything on the dashboard

Intel nuc i5 7200u 4gb mem  128gb ssd ubuntu 18.04 LTS fully updated.

**Comment by GarethSheppardCW** [comment]
Same here, I can launch anbox but if I install an apk and try to run it it either does nothing or tries to load something until it decides to give up and re-launch the appManager. It'll run some things like if I clock settings it takes me into that, but any installed APK's it doesn't like.

I'm running on a MacBook Pro, bootstrapping into Windows 10, then using VirtualBox to run Ubuntu 18.04.02 amd64.

**Comment by lokii86** [comment]
I cant even run any of the stock apps just literally nothing happens once the window pops up. Just can min/max the window close and  move around

**Comment by Bappy1988** [comment]
Same here. Can't click on any of the basic apps. 

Ryzen 1600, 16GB ECC memory, GeForce GTX 960, Ubuntu 18.04.2 LTS

**Comment by salvprest** [comment]
if you start anbox with this command `anbox session-manager --single-window` on a terminal (use this command before starting anbox after ubuntu reboot), than you are able to click on the apps?
what are you using, a mouse, a touchpad, a touchscreen?

**Comment by lokii86** [comment]
nuci57200u@KODI-HTPC:~$ anbox session-manager --single-window
[ 2019-04-27 13:50:41] [daemon.cpp:61@Run] Failed to connect to socket /run/anbox-container.socket: No such file or directory

**Comment by lokii86** [comment]
forgot to mention using a mouse

**Comment by salvprest** [comment]
ok this seem to me a duplicate of #939, so you should enable the debug logs with `sudo snap set anbox debug.enable=true` and post the result of `sudo /snap/bin/anbox.collect-bug-info` there.
then you should close this, best to have the same problem handled in only one place
